scripts/install_theme_vendor.sh

```diff
@@ -5,7 +5,7 @@ rm -rf "$VENDOR_DIR"
 
 # MathJax
 TEMP_DIR="$(mktemp -d -t "download-mathjax-XXXXXXXX")"
-MATHJAX_URL="https://registry.npmjs.org/mathjax/-/mathjax-2.7.5.tgz"
+MATHJAX_URL=${MATHJAX_URL:="https://registry.npmjs.org/mathjax/-/mathjax-2.7.5.tgz"}
```

Comment:
You can only set this to `${MATHJAX_URL:="https://registry.npmjs.org/mathjax/-/mathjax-2.7.5.tgz"}`, it means that if `MATHJAX_URL` is unset or empty, set it to `https://registry.npmjs.org/mathjax/-/mathjax-2.7.5.tgz` and return it. 
If in your style, it should be `MATHJAX_URL=${MATHJAX_URL:-"https://registry.npmjs.org/mathjax/-/mathjax-2.7.5.tgz"}`, it will return default value but not set it(you set it by yourself via `MATHJAX_URL=`)